src/lib/server/models.ts

```diff
@@ -61,6 +62,7 @@ const modelConfig = z.object({
 		.passthrough()
 		.optional(),
 	multimodal: z.boolean().default(false),
+	functions: z.boolean().default(false),
```

Comment:
cc: @gary149

should we rename this env var to `tools` rather than `functions` ?